ecr-login/cache/build.go

```diff
@@ -34,13 +33,12 @@ func BuildCredentialsCache(awsSession *session.Session, region string, cacheDir
 
 	if cacheDir == "" {
 		//Get cacheDir from env var "AWS_ECR_CACHE_DIR" or set to default
```

Comment:
GetCacheDir does more than what this comment explains. How about removing the comment from here?